src/containers/Sidebar/index.js

```diff
@@ -47,28 +50,69 @@ const Sidebar = (props) => {
 		return itemsList;
 	}, [navConfig])
 
+	useEffect(() => {
```

Comment:
@Aringoaway useEffect should be always on the top of the file, so please move it to line 19